staging test_app_code_venv test failing

```
    __________________ test_app_code_venv[ansible://app-staging] ___________________
    [gw0] linux -- Python 3.9.2 /home/sdci/securedrop-source/.venv/bin/python
    
    host = <testinfra.host.Host ansible://app-staging>
    
        def test_app_code_venv(host):
            """
            Ensure the securedrop-app-code virtualenv is correct.
            """
            cmd = f"""test -z $VIRTUAL_ENV && . {sdvars.securedrop_venv}/bin/activate && "
            "test "$VIRTUAL_ENV" = "{sdvars.securedrop_venv}" """
        
            result = host.run(cmd)
    >       assert result.rc == 0
    E       assert 127 == 0
    E        +  where 127 = CommandResult(command=b'test -z $VIRTUAL_ENV && . /opt/venvs/securedrop-app-code/bin/activate && "\n    "test "$VIRTUAL_ENV" = "/opt/venvs/securedrop-app-code" ', exit_status=127, stdout=None, stderr=b"Warning: Permanently added '192.168.121.101' (ECDSA) to the list of known hosts.\r\nbash: line 1: $'\\n    test': command not found\n").rc
```

https://github.com/freedomofpress/securedrop/actions/runs/11266722861/job/31359573205

--- a/molecule/testinfra/app/test_appenv.py
+++ b/molecule/testinfra/app/test_appenv.py
@@ -76,8 +76,10 @@ def test_app_code_venv(host):
     """
     Ensure the securedrop-app-code virtualenv is correct.
     """
-    cmd = f"""test -z $VIRTUAL_ENV && . {sdvars.securedrop_venv}/bin/activate && "
-    "test "$VIRTUAL_ENV" = "{sdvars.securedrop_venv}" """
+    cmd = (
+        f"test -z $VIRTUAL_ENV && . {sdvars.securedrop_venv}/bin/activate && "
+        + f'test "$VIRTUAL_ENV" = "{sdvars.securedrop_venv}" '
+    )
 
     result = host.run(cmd)
     assert result.rc == 0

